ci/build-project.sh

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env bash
```

Comment:
Consider stealing https://github.com/improbable/svp/blob/master/workflow/lib/setup.sh and log.sh and tools.sh and using them. They augment bash very nicely.